.env.test

```diff
@@ -5,12 +5,11 @@ SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
 PANTHER_ERROR_SCREENSHOT_DIR=./var/error-screenshots
 
-# Root access needed to create the new TEST database
-DATABASE_URL="sqlite:///%kernel.project_dir%/data/database.sqlite"
-
-TYPESENSE_URL=http://localhost:8108
+TYPESENSE_URL=http://typesense:8108
 TYPESENSE_KEY=xyz
 
 # Disable the proxy for the test env
 KOBO_PROXY_USE_EVERYWHERE=0
 KOBO_PROXY_ENABLED=0
+
+DATABASE_URL="mysql://root:biblioteca@db:3306/biblioteca?serverVersion=mariadb-10.3.39&charset=utf8"
```

Comment:
The `serverVersion` format changed with DBAL 4.x, use `serverVersion=10.3.39-MariaDB`
See https://symfony.com/doc/current/reference/configuration/doctrine.html